test-harness/index.ts

```diff
@@ -6,35 +6,47 @@ import * as fs from 'fs';
 import * as tmp from 'tmp';
 import { applyReplacements, normalizeLineEndings, parseScenarioFile, tmpFile } from './helpers';
 import { spawnNode } from './spawn';
+import nanoid = require('nanoid/non-secure');
```

Comment:
why not import from? 🤔

Reply:
doesn't work unfortunately, would need to enable default module interop which I don't want to do.

Reply:
doesn't work unfortunately, would need to enable default module interop which I don't want to do.